app.ts

```diff
@@ -0,0 +1,319 @@
+	let infoColumns: HTMLElement | null = document.getElementById("info-columns-container"); // Information
+	// Creates the row that the info will display and adds it to the infoColumnsArea.
+	let infoDataRow = `<div id="info-row-${columnData[0]}" class="info-rows"></div>`;
+
+	if (infoColumns !== null) {
+		infoColumns.innerHTML += infoDataRow;
+		// Gets the created rows.
+		let finalInfoDataRow = document.getElementById("info-row-" + columnData[0]);
+		if (finalInfoDataRow !== null) {
+			// Loops through
+			for (let x of columnData) {
+				let infoData = `<p class="info-row-data">${x}</p>`;
+				finalInfoDataRow.innerHTML += infoData;
+			}
+		}
+	}
+}
+
+function resizeScreenData() {
+	let toNumber: number;
+	let recordNav: HTMLElement | null = document.getElementById("record-navigation-container");
+	let nextBtn: HTMLElement | null = document.getElementById("next-records-btn");
+	let previousBtn: HTMLElement | null = document.getElementById("previous-records-btn");
+	let navBtns = document.getElementById("navigation-btns");
+	if (recordNav !== null) {
+		if (recordNav.contains(navBtns as HTMLDivElement)) {
+			let numberOfRows = Math.floor(window.innerHeight / 50);
+
+			(nextBtn as HTMLButtonElement).disabled = false;
+			(previousBtn as HTMLButtonElement).disabled = false;
+
+			let finalRecord = recordNumberTotal - 1;
+
+			if (fromNumber + numberOfRows >= finalRecord) {
+				fromNumber = finalRecord - numberOfRows;
+				(nextBtn as HTMLButtonElement).disabled = true;
+				(previousBtn as HTMLButtonElement).disabled = false;
+			} else if (fromNumber <= 0) {
+				(nextBtn as HTMLButtonElement).disabled = false;
+				(previousBtn as HTMLButtonElement).disabled = true;
+				fromNumber = 0;
+			}
+
+			toNumber = fromNumber + numberOfRows;
+
+			getRecords(fromNumber, toNumber);
+		}
+	}
+}
+
+window.addEventListener("resize", debounce(resizeScreenData, 500));
+
+class pageNavigation {
+	nextBtn: HTMLElement | null;
+	previousBtn: HTMLElement | null;
+	firstPageBtn: HTMLElement | null;
+	lastPageBtn: HTMLElement | null;
+	confirmationBtn: HTMLElement | null;
+
+	constructor() {
+		this.nextBtn = document.getElementById("next-records-btn");
+		this.previousBtn = document.getElementById("previous-records-btn");
+		this.firstPageBtn = document.getElementById("first-page-btn");
+		this.lastPageBtn = document.getElementById("last-page-btn");
+		this.confirmationBtn = document.getElementById("confirmation-btn");
+
+		if (this.confirmationBtn !== null) {
+			this.confirmationBtn.addEventListener("click", recordSelection);
+		}
+
+		if (this.nextBtn !== null && this.previousBtn !== null && this.firstPageBtn !== null && this.lastPageBtn !== null) {
+			let nextPage = () => {
+				let numberOfRows = Math.floor(window.innerHeight / 50);
+				fromNumber = fromNumber + numberOfRows * count;
+				let toNumber = fromNumber + numberOfRows;
+
+				let finalRecord = recordNumberTotal - 1;
+
+				(this.previousBtn as HTMLButtonElement).disabled = false;
```

Comment:
You have already checked for the `null`, so no need to hard cast. You probably want to change the type of the variable to `HTMLButtonElement` where you defined it to allow this code to keep compiling.